sentences/cs/_common.yaml

```diff
@@ -7,11 +7,60 @@ responses:
     no_device_class: "{{ area }} neobsahuje {{ device_class }}"
     no_entity: "Žádné zařízení ani entita se nejmenuje {{ entity }}"
     handle_error: "Při zpracování došlo k neočekávané chybě."
-lists: {}
+lists: 
+  color:
+    values:
+      - in: "bíl(á|ý|é|ou)"
+        out: "white"
+      - in: "čern(á|ý|é|ou)"
+        out: "black"
+      - in: "červen(á|ý|é|ou)"
+        out: "red"
+      - in: "oranžov(á|ý|é|ou)"
+        out: "orange"
+      - in: "žlut(á|ý|é|ou)"
+        out: "yellow"
+      - in: "zelen(á|ý|é|ou)"
+        out: "green"
+      - in: "modr(á|ý|é|ou)"
+        out: "blue"
+      - in: "fialov(á|ý|é|ou)"
+        out: "purple"
+      - in: "hněd(á|ý|é|ou)"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "(celsius | celsia)"
+      - in: "c"
+        out: "Celsia"
+      - "(fahrenheit | fahrenheita)"
+      - in: "f"
+        out: "Fahrenheita"
 expansion_rules:
+  area: "{area}"
+  name: "{name}"
   zapnout: "(zapni|zapnout)"
-  vypnout: "(vypni|vypnout)"
+  vypnout: "(vypni|vypnout|zastavit|zastav)"
   rozsvitit: "(rozsviť|rozsvítit)"
   zhasnout: "(zhasni|zhasnout)"
+  nastav: "(nastav|nastavit)"
+  teplota: "{temperature} stupňů {temperature_unit}"
+  brightness: "{brightness} [procent]"
+  zavri: "(zavři | zavřít)"
+  otevri: "(otevři | otevřít)"
+  garaz: "garáž"
```

Comment:
trivial expansion is not needed, also please let's not add unused expansions. To explain why - Your PR is about HassTurnOn/Off, if somebody starts working on curtain, he will add his expansion rules, which may be different from the ones you entered. It also helps to keep PR small.

Reply:
Point taken, but I have all sentences done and now working on tests. That's why there are here. Will post test Giles soon for other domains.

Reply:
Okay, but it is difficult to review unused expansion rules. For garáž (garage), I would say it is a cover device or an area. 
Of course I cannot know for sure, if I don't see the usage.

Reply:
Ok, cleaned.